{{cookiecutter.github_repository}}/settings/production.py

```diff
@@ -107,6 +107,7 @@
 # ------------------------------------------------------------------------------
 # Raises ImproperlyConfigured exception if DATABASE_URL not in os.environ
 DATABASES['default'] = env.db("DATABASE_URL")
+{% if cookiecutter.postgis == 'y' %}DATABASES['default']['ENGINE'] = 'django.contrib.gis.db.backends.postgis'{% endif %}
```

Comment:
Here let's just use:

```
DATABASES['default'].update(env.db("DATABASE_URL"))
```
This will also fix the issue of `DATABASES['default']['ATOMIC_REQUESTS'] = True` getting overwritten here.